MetadataMigration/src/main/java/com/rfs/MetadataMigration.java

```diff
@@ -60,7 +60,7 @@ public static class Args {
         public String s3Region;
 
         @ParametersDelegate
-        public ConnectionDetails.TargetArgs targetArgs;
+        public ConnectionDetails.TargetArgs targetArgs = new ConnectionDetails.TargetArgs();
```

Comment:
@chelma, is this part of the change you're doing?

Reply:
Yeah, I just pushed out a PR that does the same thing.